.devcontainer/Dockerfile

```diff
@@ -3,12 +3,19 @@ FROM python:3.10
 WORKDIR /usr/src/calitp
 
 RUN apt-get update
-RUN apt-get install -y ruby-full && gem install bundler
-RUN python -m pip install --upgrade pip
 
+# Install rvm (https://rvm.io/rvm/install#install-gpg-keys)
+RUN gpg --keyserver keyserver.ubuntu.com --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+RUN curl -sSL https://get.rvm.io | bash -s stable
+RUN echo 'source /usr/local/rvm/scripts/rvm' >> /etc/bash.bashrc
```

Comment:
I've added `rvm` to the global bash profile so that when the dev container mounts, it'll be available for use in the future.